### DISCUSSION TITLE
Auto user creation and config update

### DISCUSSION THREAD

**Discussion by divyn10** [nginx/docker-nginx-unprivileged#198] [Q&A] [answered]
Is there any support for new user creation (via some script or api) and auto config update present in `docker-nginx-unprivilidged` image.
I want to create a user and reload the nginx to pick that new user.

**Comment by alessfg** [comment, marked as answer]
Hey @divyn10, 

There is not, nor is it planned. Depending on your workflow, you could potentially first create a new user and then build the image using the Dockerfiles in this repo using the UID and GUI build args to tweak the user to suit your needs.
